pgtrigger/installation.py

```diff
@@ -9,6 +9,13 @@
 from pgtrigger import registry
 from pgtrigger import utils
 
+try:
+    from psycopg import Connection as Psycopg3Connection
+
+    HAVE_PSYCOPG3 = True
+except ImportError:
+    HAVE_PSYCOPG3 = False
```

Comment:
FYI using the `HAVE_` pattern because it's easier to add type hints later: https://adamj.eu/tech/2021/12/29/python-type-hints-optional-imports/